static/docs/commands-reference/destroy.md

```diff
@@ -39,31 +39,33 @@ $ echo foo > foo
 $ dvc add foo
 $ ls -a
 
-.dvc .git foo foo.dvc
+.dvc .git code.py foo foo.dvc
```

Comment:
where does it come from? I mean code.py

Reply:
A random code file in the folder. Just to show that the original data file and the other files are not deleted.

Reply:
if needed, `code.py` can be removed from this example.
Also, it is present in the previous example. I didn't alter the previous example.